cmd/limactl/edit.go

```diff
@@ -30,6 +32,11 @@ func newEditCommand() *cobra.Command {
 		GroupID:           basicCommand,
 	}
 	editflags.RegisterEdit(editCommand)
+
+	if runtime.GOOS != "windows" {
+		editCommand.Flags().Bool("start-at-login", false, "register(--start-at-login=true)/unregister(--start-at-login=false) the systemd/launchd unit file for instance")
```

Comment:
This should be in `pkg/editutil`

Reply:
Sorry, meant editflags

https://github.com/lima-vm/lima/blob/master/cmd/limactl/editflags/editflags.go